pkgs/by-name/ve/vesktop/package.nix

```
@@ -83,9 +86,7 @@ stdenv.mkDerivation (finalAttrs: {
     makeWrapper
   ];
 
-  patches = [
-    (substituteAll { inherit vencord; src = ./use_system_vencord.patch; })
-  ];
+  patches = lib.optionals withSystemVencord (substituteAll { inherit vencord; src = ./use_system_vencord.patch; });
```

I'm surprised this worked. I think this should either be `lib.optional` or wrap the optional element in a list.

```suggestion
  patches = lib.optionals withSystemVencord [ (substituteAll { inherit vencord; src = ./use_system_vencord.patch; }) ];
```

---

nice catch! thanks :)